app/profile/createTask/createTask.scss

```diff
@@ -1,39 +1,27 @@
-.create_task_page_container {
+.create-task {
   display: flex;
   flex-direction: column;
   align-items: center;
-}
-
-.title_container {
-  display: flex;
-}
-.title_container h2 {
-  margin-right: $margin-m;
-  margin-top: 0px;
-  margin-bottom: 0;
-}
-
-.error_message {
-  position: absolute;
-  bottom: -16px;
-}
-
-@media (max-width: $breakpoint-xl) {
-  .create_task_page_container {
-    width: 70%;
-  }
-  .title_container {
-    flex-direction: column;
-    align-items: center;
+  .create-task__title-container {
+    display: flex;
+    .create-task__title {
+      margin-right: $margin-m;
+      margin-top: 0px;
+      margin-bottom: 0;

```

Comment:
Po pierwsze, jeżeli ustawiamy 0 to nie trzbea dawać px.
Po drugie, fajnie byłoby to skrócić do postaci shorthanda z czterema wartosciami